_sass/elements/_buttons.scss

```diff
@@ -150,6 +150,10 @@ button,
     font-size: 1.9rem;
     padding: 1.5rem 3rem;
   }
+
+  &.usa-text-capitalize {
+    text-transform: capitalize;
```

Comment:
Yes I can. I was also thinking that if we pass an object to `Routes` at some point then we can use the same property that builds the `Nav` labels to populate the text in the button, eliminating the need for JS to do title casing for us.